mobile/src/main/java/ch/epfl/sdp/mobile/ui/i18n/LocalizedStrings.kt

```diff
@@ -103,6 +104,10 @@ interface LocalizedStrings {
 
   val playOnlineGames: String
 
+  val puzzlePlayingAs: String
+  val puzzleUnsolvedPuzzles: String
+  val puzzleListCD: String
```

Comment:
We used `ContentDescription` in other places, so I suggest we stick to the long version to remain coherent.

Reply:
[6e0d4bb](https://github.com/epfl-SDP/android/pull/300/commits/6e0d4bba0ab3c1f19ecc2bc120e7619c27b9fb2c)